tests/dashbio_demos/app_needle_plot.py

```diff
@@ -103,108 +103,258 @@ def description():
+                                            id='needle-mutdata-file-upload',
+                                            className='needle-upload',
+                                            children=html.Div([
+                                                'Drag and drop or ',
+                                                html.A('select files')
+                                            ]),
                                         ),
                                         html.Div(
-                                            id='needle-protein-domains-select-div',
-                                            title='If checked, it will allow the user to load '
-                                                  'mutation data such as the protein coordinate '
-                                                  '(x), mutation number (y) and mutation type '
-                                                  '(mutationGroups), individually from the protein'
-                                                  ' domains',
-                                            className='needle-dataset-header-div',
-                                            children=dcc.Checklist(
-                                                id='needle-protein-domains-select-checklist',
-                                                options=[
-                                                    {
-                                                        'label': 'Load protein domains '
-                                                                 'individually',
-                                                        'value': INDIV_DOMS_KEY
-                                                    },
-                                                    {
-                                                        'label': 'Load protein domains from '
-                                                                 'UniProt only',
-                                                        'value': UNIPROT_DOMS_KEY
-                                                    }
-                                                ],
-                                                values=[]
-                                            )
+                                            id='needle-mutdata-file-info-div'
+                                        )
+                                    ]
+                                ),
+                                html.Div(
+                                    id='needle-domain-file-div',
+                                    title='Protein data files accepted here can be two'
+                                          ' fold : \n'
+                                          '- an array of json objects with the required '
+                                          'fields "name" and "coord", which are a string '
+                                          'and a string  detailing the start and end of '
```

Comment:
Same comment / question as above.

Reply:
I don't understand if "23-34" is a value example for "name" or "coord" (or both?).

Reply:
Resolving in #293.